config/Shared/config_default.php

```diff
@@ -189,6 +191,8 @@
     'threedssvc.pay1.de', // trusted Payone domain
     'www.sofort.com', // trusted Payone domain
 ]);
+$config[KernelConstants::DOMAIN_WHITELIST][] = '*.bazaarvoice.com';
```

Comment:
just add to the list above

Reply:
the same as in the guide https://docs.spryker.com/docs/pbc/all/ratings-reviews/202212.0/third-party-integrations/integrate-bazaarvoice.html#add-the-bazaarvoice-domain-to-your-allowlist